sub-rat3/microscopy/sub-rat3_sample-data10_SEM.json

```
@@ -0,0 +1,11 @@
+{
+    "PixelSize": 0.1,
+    "FovWidth": 74,
+    "FovHeight": 76,
+    "Species": "rat",
+    "BodyPart": "CSPINE",
+    "SampleFixation": "3% paraformaldehyde - 3% glutaraldehyde",
```

```suggestion
    "SampleFixation": "3% paraformaldehyde, 3% glutaraldehyde",
```
you can apply to the other json files 😊